Functions/xASL_quant_SliceTiming.m

```diff
@@ -111,6 +112,43 @@
 	SliceTiming = (0:1:(nSlices-1)) * x.Q.SliceReadoutTime;
 elseif length(x.Q.SliceReadoutTime) == nSlices
 	SliceTiming = x.Q.SliceReadoutTime;
+	
+	% Check for SliceTiming consistency between slices if a vector was provided
+	if nSlices > 1
```

Comment:
The nomenclature should be clear throughout this function:

`SliceTiming` == absolute values of slice time after 0 (BIDS)
`SliceReadoutTime` == difference of SliceTiming (xASL legacy)

So `length(unique(SliceTiming(:)))>1` should always be true, whereas below this seems not to be the case?

I see now that you use `SliceTimingDiff` instead of `SliceReadoutTime`, why not using the old term? There is no reason not to continue using the old `SliceReadoutTime` right?

Most important is that this is clearly described in the header!

Reply:
I agree that this has to be clearly described in the headers and comments.

Though my current use is mostly correct. 
`SliceTimingDiff` is a temporary variable that describes the difference between neighboring slices, has the same length as the number of slices - 1. And can be negative for Multi-Band. It is used only internally to determine if the SliceTiming is consistent and/or multi-band. 

`SliceReadoutTime` can be either a single value (positive or zero) that describes the between slice difference or has the same length as the number of slices. This makes it different from SliceTimingDiff.

This function does the job of unifying SliceReadoutTime (legacy or BIDS) to SliceTiming (BIDS). Checks if the possibly manual input is fine.

This was all described in the header, but I have improved it - is it now more clear? 

length(unique(SliceTiming(:)))>1 - yes, this should be always true - where do you think it is not?